### PR TITLE
feat: check funds before fill

### PR DESCRIPTION
**Pull request by jeroenptrs**
## What?
* Adds a flag telling the server whether it should verify that the requested address' balance is smaller than the NEO and GAS reward levels.
* Added secure env vars to handle deployment
* Moved the webpack bundle to `.gitignore`

### CODE REVIEW

File: src/components/App/addressForm.js [hunk 1]
```diff
@@ -1,100 +1,115 @@
...
 
   render() {
     let successMessage;
     if (this.state.status === faucetStatuses[3]) {
-      successMessage = this.state.wif !== ''
-        ? `Congratulations! Your address is ${this.state.generatedAddress} and your WIF is ${this.state.wif}, look for the following tx: ${this.state.tx}. Happy developing!`
-        : `Congratulations, look for the following tx: ${this.state.tx}. Happy developing!`;
+      successMessage =
+        this.state.wif !== ""
+          ? `Congratulations! Your address is ${
```
Comment (deanpress) [inline src/components/App/addressForm.js:64]: A bit opinionated, but why are we congratulating people? :smile:

Comment (jeroenptrs) [reply src/components/App/addressForm.js:64]: Ah, congratulating them on using the faucet? Which makes creating an account on privatenet lightning fast?

Something like that 😛

**Review by deanpress — COMMENTED** [review]
Looking solid cc @revett

File: README.md [hunk 2]
```diff
@@ -24,19 +24,23 @@
...
 ## Configurable environment variables
-* `NEOSCAN` is the address of your neo-local NEOSCAN instance. Defaults to `localhost:4000` which is the standard neo-scan URL in neo-local
-* `MIN_BLOCK` is the minimum amount of blocks that need to have passed before reapplying. Defaults to `5000` blocks
-* `FAUCET_WIF` is the WIF of the Faucet account. The standard value is `KxDgvEKzgSBPPfuVfw67oPQBSjidEiqTHURKSDL1R7yGaGYAeYnr`, which holds all NEO and GAS in the neo-privatenet image
-* `FAUCET_ADDRESS` is the domain where the faucet is located. Defaults to `localhost`
-* `PORT` is the port where the faucet is located. Defaults to `4002`
-* `NEO_REWARD` is the amount of NEO the faucet rewards. Defaults to `100`
-* `GAS_REWARD` is the amount of GAS the faucet rewards. Defaults to `2000`
+
+- `NEOSCAN` is the address of your neo-local NEOSCAN instance. Defaults to `localhost:4000` which is the standard neo-scan URL in neo-local
```
Comment (revett) [inline README.md:39]: Worth having these in A-Z order to ease readability?

**Review by revett — APPROVED** [review]
1 very small comment about the README that can be resolved at a later day if needed.

:cake:

File: README.md (hunk 2 above)
Comment (jeroenptrs) [reply README.md:39]: Will do this in a separate PR, thanks cap!

**Pull request merged**